### DISCUSSION TITLE
The otter.nvim plugin does not work

### DISCUSSION THREAD

**Discussion by refaelsh** [nix-community/nixvim#2242] [Q&A] [answered]
Hello all,

I have the following simple line in my config:
```nix
otter.enable = true;
```

But doing any of the following in the `:` region of Neovim does not work:
1. `:lua ottter.activate()`
1. `:lua ottter.activate`
1. `:ottter.activate()`
1. `:ottter.activate`

Please help :-)

P.S.
CC the plugin's maintainer: @PerchunPak.

**Comment by refaelsh** [comment]
The link to the maintainer @perchun does not work. I don't know why.

**Reply by MattSturgeon** [reply]
It's PerchunPak not perchun

**Comment by khaneliman** [comment, marked as answer]
Works fine for me. We also have this issue https://github.com/nix-community/nixvim/issues/2111 about making it more ergonomic.
```lua
:lua require("otter").activate()
```

**Reply by refaelsh** [reply]
Understood. Thank you :-)
